src/assets/data/share_images_url.json

```diff
@@ -1,3 +1,3 @@
 {
-    "share_image_url": "https://content.covidactnow.org/share/373-26/"
+    "share_image_url": "https://content.covidactnow.org/share/373-27/"
```

Comment:
@chasulin FYI- I generated this share URL by running the code in this PR against the latest data snapshot...  so now when we merge this we should start getting index.html pages that point at working chart URLs.  It should also make the export links in your PR work.